diff --git a/node_modules/leaflet/dist/leaflet-src.js b/node_modules/leaflet/dist/leaflet-src.js
index 9acc7da..4fcaf9a 100644
--- a/node_modules/leaflet/dist/leaflet-src.js
+++ b/node_modules/leaflet/dist/leaflet-src.js
@@ -2447,7 +2447,7 @@
   	// this method is only used for elements previously positioned using setPosition,
   	// so it's safe to cache the position for performance
 
-  	return el._leaflet_pos || new Point(0, 0);
+  	return el && "_leaflet_pos" in el && el._leaflet_pos || new Point(0, 0);
   }
 
   // @function disableTextSelection()
